sharkfuser/tests/CMakeLists.txt

```diff
@@ -47,3 +47,8 @@ add_sharkfuser_test(
   SRCS
     test_logging.cpp
 )
+
+add_sharkfuser_lit_test(
+	SRC example_lit_test.cpp
```

Comment:
Not too important rn but eventually we might want to have this take in multiple SRCS.

Reply:
👍, when we need to support multiple I think we would need a `SRCS` + `TEST_FILE` argument or something, otherwise I'm not sure how the function would know which file contained the lit directives.

Reply:
Let's just do this: https://sourcegraph.com/github.com/iree-org/iree/-/blob/build_tools/cmake/iree_lit_test.cmake?L102